### ISSUE TITLE
bug converting cDNA to genomic position with Mutalyzer API v3 

### ISSUE THREAD

**Issue by lmatalonga** [mutalyzer/normalizer#43]
Hi team, 

We are trying to use your API to convert cDNA sequenced to genomic positions: https://v3.mutalyzer.nl/positionconverter?referenceId=NM_000334.4&fromSelectorId&fromCoordinateSystem=c&position=9877&toSelectorId&toCoordinateSystem=g&includeOverlapping=true 

The results obtain is not valid with this version of the API, in this example it should be: NC_000017.10:g.62013765C>T as we correctly obtain when using Mutalyzer v2: https://mutalyzer.nl/position-converter?assembly_name_or_alias=GRCh37&description=NM_000334.4%3Ac.9877G%3EA

Thanks!
Leslie Matalonga 

-- 
Leslie Matalonga, PhD
Clinical Genomics Specialist
CNAG-CRG
Tel:934020828

**Comment by jfjlaros** [comment]
I think the lack of documentation is the main problem here. In your query, a conversion from `NM_000334.4:c.` to `NM_000334.4:g.` is requested.

You probably wanted something like [this](https://v3.mutalyzer.nl/api/position_convert/?reference_id=NC_000017.10&from_selector_id=NM_000334.4&from_coordinate_system=c&position=9877&to_coordinate_system=g&include_overlapping=false).

I will leave this issue open to make sure we add proper documentation.

**Comment by lmatalonga** [comment]
Thank you very much for the quick reply! Yes, the conversion requested is from NM_000334.4:c. to NM_000334.4:g. but we do not have the reference ID (NC_000017.10) that seems to be mandatory in v3 and not in v2? 

Our input is NM_000334.4:c.9877G>A for GRCh37.

**Comment by jfjlaros** [comment]
Indeed, in the old version, the reference ID was given implicitly by specifying a build (GRCh37 in your case). We now have support for all builds and assemblies, with the slight downside that the reference ID must be provided.

**Comment by lmatalonga** [comment]
Thanks, we will try to see how to easily get the reference ID to overcome this. Just as a suggestion, it would be easier for use to be able to just state the version of the genome we are using (e.g. GRCh37) in the input instead of the reference ID as in our case we do not have the chr associated to the information - we would need to first get it from the gene. 

Thanks again for the support!
Leslie

**Comment by jfjlaros** [comment]
We could perhaps think about a separate endpoint that returns a reference ID, given a transcript ID and a genome build. On the other hand, I would expect such a service from the NCBI.
